src/main/resources/entity-types/finance/simple_ledger.json5

```
@@ -69,8 +76,28 @@
       },
       sourceAlias: 'ledger',
       queryable: true,
+      essential: true,
       visibleByDefault: false,
-      valueGetter: ":sourceAlias.jsonb ->> 'fiscalYearOneId'"
+      valueGetter: "(:sourceAlias.jsonb ->> 'fiscalYearOneId'::text)",
+      filterValueGetter: "(:sourceAlias.jsonb ->> 'fiscalYearOneId'::text)"
+    },
+    {
+      name: 'fiscal_year_one',
+      sourceAlias: 'ledger',
+      dataType: {
+        dataType: 'stringType',
+      },
+      idColumnName: 'fiscal_year_one_id',
+      queryable: true,
+      visibleByDefault: true,
+      essential: true,
+      valueGetter: "( SELECT fis_y.jsonb ->> 'name' FILTER (WHERE fis_y.jsonb ->> 'name' IS NOT NULL) FROM (:sourceAlias.jsonb ->> 'fiscalYearOneId') record(value) JOIN src_finance_fiscal_year fis_y ON record.value::text = fis_y.id::text )",
+      filterValueGetter: "( SELECT lower(fis_y.jsonb ->> 'name') FILTER (WHERE fis_y.jsonb ->> 'name' IS NOT NULL) FROM (:sourceAlias.jsonb ->> 'fiscalYearOneId') record(value) JOIN src_finance_fiscal_year fis_y ON record.value::text = fis_y.id::text )",
```

Just to be safe, I'd add ${tenant_id} to these source view references
```suggestion
      valueGetter: "( SELECT fis_y.jsonb ->> 'name' FILTER (WHERE fis_y.jsonb ->> 'name' IS NOT NULL) FROM (:sourceAlias.jsonb ->> 'fiscalYearOneId') record(value) JOIN ${tenant_id}_src_finance_fiscal_year fis_y ON record.value::text = fis_y.id::text )",
      filterValueGetter: "( SELECT lower(fis_y.jsonb ->> 'name') FILTER (WHERE fis_y.jsonb ->> 'name' IS NOT NULL) FROM (:sourceAlias.jsonb ->> 'fiscalYearOneId') record(value) JOIN ${tenant_id}_ssrc_finance_fiscal_year fis_y ON record.value::text = fis_y.id::text )",
```